homeassistant/components/pvoutput/sensor.py

```diff
@@ -66,10 +72,14 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class PvoutputSensor(SensorEntity):
     """Representation of a PVOutput sensor."""
 
+    _attr_state_class = STATE_CLASS_MEASUREMENT
+    _attr_device_class = DEVICE_CLASS_ENERGY
+    _attr_unit_of_measurement = ENERGY_WATT_HOUR
```

Comment:
Without last reset, we are not building up min/max/mean of energy sensors?

I think state_class is not correct here, as this value does not represent a current measurement.